src/frame/js/smooch.jsx

```diff
@@ -216,6 +216,14 @@ export function login(userId = '', jwt, attributes) {
             }
         }
     })).then((loginResponse) => {
+        Raven.setUserContext({
+            id: loginResponse.appUser.userId || loginResponse.appUser._id
+        });
+
+        Raven.setExtraContext({
+            appToken
```

Comment:
we'll replace this with appId when it will be used. I thought I could do it right now, but we don't return the appId as part of the init response.

This will be useful to track who's getting errors and get in touch if necessary.